backend/src/main/java/com/redhat/cloud/notifications/db/Query.java

```diff
@@ -1,9 +1,16 @@
 package com.redhat.cloud.notifications.db;
 
+import javax.ws.rs.BadRequestException;
 import javax.ws.rs.QueryParam;
 import java.util.function.Function;
+import java.util.regex.Pattern;
+
+import static java.util.regex.Pattern.CASE_INSENSITIVE;
 
 public class Query {
+
+    private static final Pattern SORT_FIELD_PATTERN = Pattern.compile("^[a-z0-9_-]+$", CASE_INSENSITIVE);
```

Comment:
@josejulio The backend will no longer allow sorting on fields that do not match this pattern. Please confirm that this won't break anything in the frontend.

Reply:
IIRC there is at least one place where we use `something.created`. I'll try to search what endpoint is calling.

Reply:
`/api/integrations/v1.0/endpoints/{id}/history` uses: `sortBy: 'nh.created:desc'`

Reply:
Good catch! I added `.` to the allowed characters in the regex.